### PR TITLE
SDK for Metrics and XAI

### PR DESCRIPTION
**Pull request by sinnec**
closes: https://github.com/squaredev-io/administration/issues/466

### CODE REVIEW

File: whitebox/sdk/whitebox.py
Comment (sinnec) [reply whitebox/sdk/whitebox.py:240]: Actually if there are no metrics but the model exists, it returns an empty list!

File: whitebox/sdk/whitebox.py
Comment (sinnec) [reply whitebox/sdk/whitebox.py:254]: Same here!

**Comment by sinnec** [comment]
@momegas Thanks for the review, but please only approve the review when no chnges are needed, since you'll need to re-approve it after the changes in order to merge. Thanks! 🙏

**Review by momegas — APPROVED** [review]

**Pull request merged**
